graphbolt/src/csc_sampling_graph.cc

```diff
@@ -123,11 +123,42 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::InSubgraph(
 
 c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::SampleNeighbors(
     const torch::Tensor& nodes) const {
-  // TODO(#5692): implement this.
+  const int64_t num_nodes = nodes.size(0);
```

Comment:
`num_seeds` to distinguish with `g.NumNodes()`?

Reply:
good suggestion, let me change.